Source/Renderers/DownASTRenderable.swift

```diff
@@ -22,6 +23,21 @@ extension DownASTRenderable {
     public func toAST(_ options: DownOptions = .default) throws -> UnsafeMutablePointer<cmark_node> {
         return try DownASTRenderer.stringToAST(markdownString, options: options)
     }
+
+    /// Generates an abstract syntax tree from the `markdownString` property
```

Comment:
Please update this comment doc to be accurate (it speaks about the AST rather than a `Document`)

Reply:
Not entirely wrong, since the document contains the whole AST. But updated the comment to make it clearer.